docs/community-ee/community-ee-announcement.md

```diff
@@ -115,4 +96,62 @@ Ansible Community Execution Environment Minimal <ansible-core-version-minimal-ee
 Get the details of both the images [here](<link to the forum post>).
 
 On behalf of the Ansible community, thank you and happy automating!
+
 ```
+
+
+## Bullhorn Announcement
+
+We’re happy to announce the release of the Ansible Community Execution Environment Minimal <version> (Latest) and Ansible Community Execution Environment Base <version> are here.
+
```

Comment:
Consider if you want all the details in the Bullhorn, or if you want to trim it down and just add a link to the forum post for the release. Saves you (or whoever is doing this) some repeat typing. My suggestion would be the details go in the forum topic, and everything else is a short sentence or  two and a pointer to that post.

Reply:
Edited the Bullhorn announcement. Does it look good?

Reply:
I think you can remove https://github.com/ansible-community/images/pull/92/files#diff-f529338b0d6703abb57812ebed7300a6991e77c02ec2d4baece6a8463b44b433R111-R131 (that is, everything after the new link to the forum post you added.  All that is covered in the forum and imo doesn't need to be repeated in the bullhorn.

Reply:
@anweshadas please take a look at this comment as I don't think you replied or made the suggested changes.. thanks!